docs/development_guide/modules.md

```diff
@@ -18,6 +18,8 @@ Additionally, we extend it with a few quality of life improvements.
     ├── docs/
     │   └── extra_docs.md
     ├── resources/
+    │   ├── user_input/
+    │   │   └── user_shapes.geojson
     │   ├── download1.nc
     │   └── download2.csv
```

Comment:
```suggestion
    │   ├── user/
    │   │   └── user_shapes.geojson
    │   ├── automatic/
    │   │   └── download1.nc
    │   │   └── download2.csv
```

Reply:
I had to do a separate commit because there are some lines that break the tree in this suggestion.